models/demos/llama3/README.md

```diff
@@ -0,0 +1,58 @@
+# Llama3 Models
+
+This codebase includes the Llama3 family of models.
+
+The current version supports the following Llama3 models:
+- Llama3.1-8B
+- Llama3.2-1B
+- Llama3.2-3B
+
+All the above llama models are compatible and tested on the following Tenstorrent hardware:
+- N150 (1-chip)
+- N300 (2-chips)
+- T3000 (8-chips)
+
+## How to Run
+
+### Download the weights
+
+Download the weights [directly from Meta](https://llama.meta.com/llama-downloads/), this will mean accepting their license terms.
+
+The downloaded directories include weight files (e.g. `consolidated.00.pth`), the tokenizer `tokenizer.model` and configuration file `params.json`.
+
+### Setup TT environment
+
+1. Set up environment variables:
+```
+export LLAMA_DIR=<meta_llama_model_dir>
+export WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml
```

Comment:
Cannot be made optional since we rely on full grid of 8x8.

The LLAMA_DIR also needs to be there since this codebase works across all llama3 models, and you need to specify in some way which model you want.

I've added a description for both variables 👍

Reply:
It should be optional for one row harvested WH

Reply:
I'll add a comment mentioned that for N150 this is optional then.

But I'll leave the exports as is since it covers all WHs

Reply:
I've updated the description to: 

- `$WH_ARCH_YAML` sets the dispatch over ethernet cores. This is optional for N150 and required for N300 and T3000, enabling a full core grid utilization (8x8), allowing for maximum performance of LLama3 models.